apkid/rules/apk/protectors.yara

```diff
@@ -88,3 +88,20 @@ rule vkey_apk : protector
     1 of ($asseta*) and
     1 of ($assetb*)
 }
+
+rule free_rasp : protector
+{
+  meta:
+    description = "FreeRASP"
+    url         = "https://www.talsec.app/freerasp-in-app-protection-security-talsec"
+    sample      = "2b8faa038bf34474075a56e2fda7887a7df9c3c57db8a9f25547dc9374137ec9"
+    author      = "Fare9"
+
+  strings:
+    $lib1   = /lib\/(arm.*|x86.*)\/libsecurity\.so/
+    $lib2   = /lib\/(arm.*|x86.*)\/libpolarssl\.so/
+    $asset = "assets/talsec"
```

Comment:
@Fare9 can you balance the whitespaces?

Reply:
If you remove the file `assets/talsec`, will we have false positives on apps having these two native libraries?